packages/cosmic-swingset/Makefile

```diff
@@ -143,12 +143,9 @@ scenario2-run-chain-economy: t1/decentral-economy-config.json
 	CHAIN_BOOTSTRAP_VAT_CONFIG="$$PWD/t1/decentral-economy-config.json" \
 		$(MAKE) scenario2-run-chain
 
-scenario2-run-chain-psm: ../vats/decentral-psm-config.json
-	CHAIN_BOOTSTRAP_VAT_CONFIG="$$PWD/../vats/decentral-psm-config.json" \
-		$(MAKE) scenario2-run-chain
-
-scenario2-run-chain:
-	OTEL_EXPORTER_PROMETHEUS_PORT=$(OTEL_EXPORTER_PROMETHEUS_PORT) \
+scenario2-run-chain: ../vats/decentral-devnet-config.json
+	CHAIN_BOOTSTRAP_VAT_CONFIG="$$PWD/../vats/decentral-devnet-config.json" \
```

Comment:
a little surprising to have this default to devnet. consider a comment to call it out and say why